govdelivery-subscribe/error/index.html

```diff
@@ -11,13 +11,13 @@
 {% block content_main %}
     <h1><span class="cf-icon cf-icon-error-round"></span> Not just yet!</h1>
     <p class="short-desc">
-        We’re still building the email signup function for <br>
-        this beta release. It will be available soon. For now:
+        We couldn’t complete your signup request. Try<br>
```

Comment:
I think I agree with Ans on this. It just seems unnecessary to put someone through the pain of leaving the signup page in the first place.

Though another question would be: do we validate that it's structured as an email address? If we do, then it seems even easier to not let them submit in the first place. I don't know if GovDelivery validates—a user has to click to verify, so they may not care about the submission structure—but if they do we might have to serve the error then, right?